meta/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 galaxy_info:
+  role_name: nexus_role
+  namespace: idealista
   company: Idealista S.A.
```

Comment:
The company name is Idealista S.A.U.

Could you also reflect here that the role supports Debian 11 in the `platforms` section?

Reply:
Fixed :)